### DISCUSSION TITLE
Delay compensation BUG

### DISCUSSION THREAD

**Discussion by Ardalien** [apohl79/audiogridder#368] [Help]
Hello. My regards for getting to stable 1.1.0. The Fab Filter Pro Q2 Delay compensation problem is still there :(. So to get back to the problem the Fab Filter Pro Q2 it has 3 operating modes. Zero Latency, Dynamic Phaze and Linear Phase( in Linear Phase there are these options: min,med,high,max,ultra) If I load it up in Audiogridder plugin the plugin stats operating in Zero Latency. It works perfectly, delay compensation correctly compensated. When I select in linear phase mode (Which is a cpu hogger sometimes but very useful) the plugin compensates correctly. Then I save the project and close it. When I reopen the project( In Ableton 10)  The delay from the audiogridder plugin ( only the ones with FF Pro Q2 ) is bigger then 400 ms. After that I bypass the Pro Q2 inside the audiogridder plugin and eneble it again it works correctly. Or I select The operating mode to Zero Lantency and Back to Linear Phase and it works correctly. The thing is I tried to map an automation on the bypass and/or also Host bypass to achieve the bypass and make it delay compensate correctly so I don t have to search thru each individual audiogridder plugin to do it manually.. I also tried to make a key automation to the Proccesing mode to go to Zero Latency than Back to linear from the key I mapped. I Doesn't work ONLY if I click with the mouse in the audiogridder plugin to bypass or if I switch the proccesing mode , again to Zero Latency and back to Linear Phase with the mouse on the screen. Until you find a solution for this can you make the bypass switch from the Audiogridder plugin mappable, so I can map all the instances to one knob to resolve this faster ? :) 
Thank you and good luck.

**Comment by apohl79** [comment]
Would you be able to create log and trace files of the server and the plugin:

1) when you add the plugin in the first place to AG with zero latency, and then changing (so I see how server and plugin are updating the latency samples) and
2) when you reload the session and fix it by bypassing/unbypassing the fab filter plugin

I f possible, rename the files to one of the two steps, so I know what I have to look for.

Thanks a lot!

PS: Here is the guide for providing those files: https://audiogridder.com/bug-reports/

**Reply by Ardalien** [reply]
Here are The Trace Files from the Server on Windows I hope It helps you find the problem.
[Archive.zip](https://github.com/apohl79/audiogridder/files/5988439/Archive.zip)

**Reply by Ardalien** [reply]
Maybe I forgot to mention, I don t know if it is important, but I am not using screen capture  I am tweaking the plugin from the Server.
